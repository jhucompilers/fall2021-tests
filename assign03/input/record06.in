PROGRAM record06;
  VAR x : RECORD
    x : RECORD
      x : RECORD
        x : INTEGER;
      END;
    END;
  END;
BEGIN
    READ x.x.x.x;
END.