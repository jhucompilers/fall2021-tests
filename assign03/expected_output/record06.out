3,VAR,x,INTEGER
2,VAR,x,RECORD (INTEGER)
1,VAR,x,RECORD (RECORD (INTEGER))
0,VAR,x,RECORD (RECORD (RECORD (INTEGER)))
